sql/core/src/test/resources/sql-tests/results/pgSQL/aggregates_part2.sql.out

```diff
@@ -0,0 +1,320 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 28
+
+
+-- !query 0
+-- boolean and transitions
+     -- null because strict
+     -- and actual computations
+
+     -- boolean or transitions
+     -- null because strict
+     -- actual computations
+
+
+
+
+
+
+
+
+explain
+  select min(unique1) from tenk1
+-- !query 0 schema
+struct<plan:string>
+-- !query 0 output
+== Physical Plan ==
+*HashAggregate(keys=[], functions=[min(unique1#x)])
++- *HashAggregate(keys=[], functions=[partial_min(unique1#x)])
+   +- *Project [unique1#x]
+      +- *BatchScan[unique1#x] CSVScan Location: InMemoryFileIndex[file:/Users/yumwang/SPARK-27883/spark/sql/core/target/scala-2.12/test-classes/t..., ReadSchema: struct<unique1:int>
```

Comment:
Do we need `explain`? It contains the file path when `explain`.

Reply:
we do not need it.

Reply:
Done